spec/enum_spec.rb

```diff
@@ -0,0 +1,202 @@
+# rubocop:disable all
+
```

Comment:
You should not disable rubocop. Instead, you should fix the errors. I have removed that disable and a lot of errors popped up. ⬇️  Kindly fix these errors

![Screenshot from 2021-01-08 12-02-22](https://user-images.githubusercontent.com/32361963/103995557-b270ec80-51a9-11eb-99c3-93f8cbc9fb91.png)

Reply:
will be fixed as expected